[ADD] inventory: product type

Felicious opened the pull request:
### Summary
Introducing new doc that introduces:
- product types
- how the product type affects common inventory operations
  - chart that summarizes this info
  - in-depth explanation in paragraph form

Adapted from [Inventory smartclass content](https://www.odoo.com/slides/slide/products-2146?fullscreen=1#)

[Task](https://www.odoo.com/web#id=3811247&cids=3&model=project.task&view_type=form)

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/8091).

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+Set product type
+================
+
+To set a product type, go to to :menuselection:`Inventory app --> Products --> Product` and
```
```suggestion
To set a product type, go to :menuselection:`Inventory app --> Products --> Product` and
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+- :guilabel:`Consumable` are items always assumed to be in stock, but with untracked and
+  unforecasted quantities. Examples include nails, toilet paper, and coffee—replaceable, essential,
+  but exact counts are unnecessary
+- :guilabel:`Service` are products are services performed by the business that is for sale (e.g.
+  maintenance, installation, repair)
```
```suggestion
- :guilabel:`Service` products are services performed and sold by the business (e.g.,
  maintenance, installation, repair)
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+   Choose :guilabel:`Storable` if it is necessary to track a product's stock at various locations,
+   inventory valuation, or lots and serial numbers.
+
+- :guilabel:`Consumable` are items always assumed to be in stock, but with untracked and
+  unforecasted quantities. Examples include nails, toilet paper, and coffee—replaceable, essential,
+  but exact counts are unnecessary
```
```suggestion
- :guilabel:`Consumable` products are items always assumed to be in stock, but with untracked and
  unforecasted quantities (e.g., nails, toilet paper, and coffee). Consumables are replaceable and essential,
  but exact counts are unnecessary
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+Define *product types* in Odoo to determine how the database keeps track of the product.
```
I think this intro sentence could use a brief elaboration to better help the reader understand what product types are

Felicious replied:
Done! :D

```suggestion
Define *product types* in Odoo to track products in varying levels of detail.

Classify products as *storable* to track stock counts, allowing users to trigger :doc:`reordering
rules <../product_replenishment/reordering_rules>` for generating purchase orders. *Consumable*
products are assumed to always be in stock, and *service* products are performed and served by the
business.
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+
+- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Calendar* and
+  *Appointment* apps. Requires the :guilabel:`Pay to Book` module to be installed
+- :guilabel:`Combo`: create discounted products sold in a bundle, available with the *POS* app
```
```suggestion
- :guilabel:`Combo`: create discounted products sold in a bundle, available with the *POS* app
```
Is it POS or PoS? Genuinely asking, could be worth a check

Felicious replied:
@brse-odoo Looks like in our Odoo docs, we use POS. However, it's written by the BE team, and idk how much they care about maintaining style consistency. I'll use PoS, since it more accurately represents the spelled out "Point of Sale" (: We use BoM instead of BOM, after all.

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+  app
+- :guilabel:`Event Booth`: charge customers for setting up a booth. Requires installation of the
+  *Events* app
+- :guilabel:`Course`: sell access to an educational course. Requires installation of the *eLearning*
```
Just a note - some of these bullets end with a period, some don't - I think it'd be nice to end them all the same way consistently

Felicious replied:
Thanks for pointing it out!

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+=============
+
+Below is a summary of how each product type affects common *Inventory* operations, like transfers,
+reordering rules, and the forecasted report. Click the chart item with an asterisk(*) to navigate to
```
```suggestion
reordering rules, and the forecasted report. Click the chart item with an asterisk (*) to navigate to
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+On-hand quantity
+----------------
+
+A storable product's on-hand and forecasted quantity based on incoming and outgoing orders are
```
```suggestion
A storable product's on-hand and forecasted quantities, based on incoming and outgoing orders, are
```
Since these are two separate quantities

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+
+When creating a transfer for storable products in the *Inventory* app, transfers modify the on-hand
+quantity at each location. For example, when five units of product is moved from the internal
+location `WH/Stock` to `WH/Packing Zone`, the five units recorded at `WH/Stock` is decreased, and
```
```suggestion
location `WH/Stock` to `WH/Packing Zone`, the five units recorded at `WH/Stock` is decreased, and the
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+-------------
+
+The replenishment report, accessed by going to :menuselection:`Inventory app --> Operations -->
+Replenishment` is a list of products that have a negative forecasted quantity and have manual
```
```suggestion
Replenishment` is a list of products that have a negative forecasted quantity, **and** have manual
```
Just placing emphasis on this since both conditions are needed to appear here

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+.. seealso::
+   :doc:`../product_replenishment/strategies`
+
+Only storable products appear on the replenishment report.
```
Is this important enough of a fact to get its own admonition?

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+   :align: center
+   :alt: alt text
+
+Since quantities of consumable products, users must manually enter the :guilabel:`Quantity` to order
+in the :guilabel:`Low on stock? Let's replenish` pop-up window.
```
I think this sentence needs clarification - I'm not sure what it's trying to say

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+Packaged
+--------
+
+Storable and consumable products can both be placed in packages.
```
```suggestion
Both storable and consumable products can be placed in packages.
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+.. seealso::
+   :doc:`package`
+
+However, for consumable products, the quantity is not tracked, and the product is not be listed in
```
```suggestion
However, for consumable products, the quantity is not tracked, and the product is not listed in
```

brse-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+
+   A consumable product was placed in the package, but the **Content** list does not list it.
+
+If the *Move Entire Package* feature is enabled, moving a package that contains updates the location
```
```suggestion
If the *Move Entire Package* feature is enabled, moving a package that contains updates the location
```
A package that contains what?

brse-odoo reviewed (approved):
Hi @Felicious, I've finished my review of this PR. Great job! This is a good addition to the Inventory documentation. I left suggestions (mostly for grammatical fixes) for you to accept/reject as you see fit -- so I'm approving this for the next round.

Feel free to reach out if you have any questions, thanks!

Felicious commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,217 @@
+- :guilabel:`Event Booth`: charge customers for setting up a booth. Requires installation of the
+  *Events* app
+- :guilabel:`Course`: sell access to an educational course. Requires installation of the *eLearning*
+  app.
```
```suggestion
  app
```

Felicious commented:
Hi @odoo/us-doc-review This PR is ready for final review!!

Felicious commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+   To install additional applications, navigate to :menuselection:`Apps` and select the
+   :guilabel:`Activate` button to install the desired apps or modules.
+
+- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Calendar* and
+  *Appointment* apps. Requires the :guilabel:`Pay to Book` module to be installed
```
Hi @tiku-odoo ! What do you think of these descriptions of product types, as the calendar expert?

tiku-odoo replied:
This description looks fine. I need to familiarize myself with the pay-to-book module. You're good to go here, though. Booking of appointments would be through the appointments app (also the integration with website).

Felicious replied:
```suggestion
- :guilabel:`Booking Fees`: charge a fee for booking appointments through the
  *Appointment* app. Requires the *Calendar* app and :guilabel:`Pay to Book` module to be installed
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+Set product type
+================
+
+To set a product type, go to :menuselection:`Inventory app --> Products --> Product` and selecting
```
```suggestion
To set a product type, go to :menuselection:`Inventory app --> Products --> Product`, and select
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+
+On the product form, in the :guilabel:`Product Type` field, select:
+
+- :guilabel:`Storable` for products tracked with stock counts. Only storable products can trigger
```
```suggestion
- :guilabel:`Storable Product` for products tracked with stock counts. Only storable products can trigger
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+  reordering rules for generating purchase orders
+
+.. tip::
+   Choose :guilabel:`Storable` if it is necessary to track a product's stock at various locations,
```
```suggestion
   Choose :guilabel:`Storable Product` if it is necessary to track a product's stock at various locations,
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+
+.. tip::
+   Choose :guilabel:`Storable` if it is necessary to track a product's stock at various locations,
+   inventory valuation, or lots and serial numbers.
```
```suggestion
   inventory valuations, or if the product has lots and serial numbers.
```
does that still ring true, with my rewording here?

Felicious replied:
It does! Thank you for fixing up the language here (:

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+   Choose :guilabel:`Storable` if it is necessary to track a product's stock at various locations,
+   inventory valuation, or lots and serial numbers.
+
+- :guilabel:`Consumable` products are items always assumed to be in stock, but with untracked and
```
```suggestion
- :guilabel:`Consumable` products are items that are always assumed to be in stock, but with untracked and
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+   inventory valuation, or lots and serial numbers.
+
+- :guilabel:`Consumable` products are items always assumed to be in stock, but with untracked and
+  unforecasted quantities (e.g., nails, toilet paper, and coffee). Consumables are replaceable and
```
```suggestion
  unforecasted quantities (e.g. nails, toilet paper, coffee, etc.). Consumables are replaceable and
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+- :guilabel:`Consumable` products are items always assumed to be in stock, but with untracked and
+  unforecasted quantities (e.g., nails, toilet paper, and coffee). Consumables are replaceable and
+  essential, but exact counts are unnecessary
+- :guilabel:`Service`: perform and sell services (e.g., maintenance, installation, repair)
```
```suggestion
- :guilabel:`Service`: perform and sell services (e.g. maintenance, installation, repair)
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+   The product types listed above are part of the standard *Inventory* app. For access to the fields
+   below, install the corresponding apps **in addition** to *Inventory*.
+
+   To install additional applications, navigate to :menuselection:`Apps` and select the
```
```suggestion
   To install additional applications, navigate to the :menuselection:`Apps` application, and select the
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+
+A storable product's on-hand and forecasted quantities, based on incoming and outgoing orders, are
+reflected on the product form, accessed by going to :menuselection:`Inventory app --> Products -->
+Products`.
```
```suggestion
Products`, and selecting the desired product.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+   :align: center
+   :alt: Show "On Hand" and "Forecast" smart buttons.
+
+   Current and forecasted quantities displayed in the **On Hand** and **Forecasted** smart buttons
```
```suggestion
   Current and forecasted quantities are displayed in the :guilabel:`On Hand` and :guilabel:`Forecasted` smart buttons
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+Both storable and consumable products can be included in a request for quotation in the *Purchase*
+app.
+
+However, when receiving consumable products, their on-hand quantity will not change upon validating
```
```suggestion
However, when receiving consumable products, their on-hand quantity does not change upon validating
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+   :align: center
+   :alt: Show "Bill of Materials" and "Used In" smart buttons.
+
+   When the **Bill of Materials** and **Used In** smart buttons are visible on the product form,
```
I'd throw a GUI on both of those smart button mentions because they appear on the UI -- 👍

Felicious replied:
I agree that these deserve a GUI, but the caption text doesn't render like the rest of the doc.

When I use guilabel, no bold shows up
![image](https://github.com/odoo/documentation/assets/23611711/167b3b35-0173-40d6-ae38-c6d513089470)
![image](https://github.com/odoo/documentation/assets/23611711/f2d5b3ad-c5d0-4273-aee8-5e59275446cc)

However, when I use bold, just a littleee bit of bold shows up
![image](https://github.com/odoo/documentation/assets/23611711/379a198c-794e-4839-8d62-4673c3d6d3b2)

so even though it's wrong by our guidelines, to make the terms appear bolded in the result, I'm planning to not use guilabels in the captions.

What do you think about this?

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+
+However, for consumable products, the quantity is not tracked, and the product is not listed in the
+package's :guilabel:`Contents`, accessed by going to :menuselection:`Inventory app --> Products -->
+Packages` and selecting the desired package.
```
```suggestion
Packages`, and selecting the desired package.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+   :align: center
+   :alt: Show Packages page, containing the package contents list.
+
+   A consumable product was placed in the package, but the **Content** list does not list it.
```
should "Content" have a GUI here, as well --- what are your thoughts?

Felicious replied:
Since this is another caption in the `figure` tag, the guilabel doesn't render as expected, so I decided to simply bold it (:

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+
+   A consumable product was placed in the package, but the **Content** list does not list it.
+
+If the *Move Entire Package* feature is enabled, moving a package updates the location of the
```
If I'm understanding correctly this feature is italicized because it's not immediately on the UI and is simply being used in a definition-like way. Is that right? 😄

Felicious replied:
Yes! The term is italicized because the feature is being referred to, but it's currently not displayed on the UI

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,233 @@
+   :align: center
+   :alt: Show stock reporting list found in Inventory > Reporting > Stock.
+
+The *location report* is a breakdown of each location (internal, external, or virtual) and the
+on-hand and reserved quantity of each storable product. The report is only available to users with
+:doc:`administrator access <../../../../general/users/access_rights>`, and is found by navigating to
+:menuselection:`Inventory app --> Reporting --> Locations`.
```
correct me if I'm wrong, but doesn't 'Storage Locations' need to be activated as well in order to see this report?

Felicious replied:
Great catch!!
```suggestion
The *location report* is a breakdown of each location (internal, external, or virtual) and the
on-hand and reserved quantity of each storable product. The report is only available with the *Storage Location* feature activated, and to users with
:doc:`administrator access <../../../../general/users/access_rights>`. Navigate to the location report by going to
:menuselection:`Inventory app --> Reporting --> Locations`.
```

ksc-odoo reviewed (changes requested):
hey @Felicious -- just finished my Final Review on this PR. Awesome work. Once you address the feedback I left, and make the necessary adjustments, tag me again for another quick look. Thanks! 👍

Felicious commented:
Hi @ksc-odoo! Thank you for your thorough first review of this doc. I've addressed your comments, as well as drawn your attention to that ongoing discussion we were having about the guilabel vs bolded text in the figure's caption. Let me know what you think, in your second review (:

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,236 @@
+
+- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Appointment* app.
+  Requires the *Calendar* app and :guilabel:`Pay to Book` module to be installed
+- :guilabel:`Combo`: create discounted products sold in a bundle, available with the *PoS* app
+  installed
```
```suggestion
- :guilabel:`Combo`: create discounted products sold in a bundle. Requires the installation of the *PoS* app.
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,236 @@
+  Requires the *Calendar* app and :guilabel:`Pay to Book` module to be installed
+- :guilabel:`Combo`: create discounted products sold in a bundle, available with the *PoS* app
+  installed
+- :guilabel:`Event Ticket`: sold to attendees wanting to go to an event. Requires installation of
```
```suggestion
- :guilabel:`Event Ticket`: sold to attendees wanting to go to an event. Requires the installation of
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,236 @@
+- :guilabel:`Event Ticket`: sold to attendees wanting to go to an event. Requires installation of
+  the *Events* app
+- :guilabel:`Event Booth`: sold to partners or sponsors to set up a booth at an event. Requires
+  installation of the *Events* app
```
```suggestion
  the installation of the *Events* app
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,236 @@
+  the *Events* app
+- :guilabel:`Event Booth`: sold to partners or sponsors to set up a booth at an event. Requires
+  installation of the *Events* app
+- :guilabel:`Course`: sell access to an educational course. Requires installation of the *eLearning*
```
```suggestion
- :guilabel:`Course`: sell access to an educational course. Requires the installation of the *eLearning*
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,236 @@
+Manufacture or subcontract
+--------------------------
+
+Storable and consumable products can both be manufactured, subcontracted, or included in a bill of
```
```suggestion
Storable and consumable products can be manufactured, subcontracted, or included in a bill of
```

ksc-odoo commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,236 @@
+   A consumable product was placed in the package, but the **Content** list does not list it.
+
+If the *Move Entire Package* feature is enabled, moving a package updates the location of the
+contained storable products. However, consumable products' locations are not updated.
```
```suggestion
contained storable products. However, the location of consumable products are not updated.
```

ksc-odoo reviewed (approved):
hey there @Felicious -- great work on this doc. Only have a small handful of suggestions that require your attention, but I'm gonna approve this now, because I don't anticipate you having any issues. Once this is ready, you can feel free to move it to Tech Review. Nice job! 👍

Felicious commented:
Hi @samueljlieber ! This PR is ready for your tech review (:

samueljlieber commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,236 @@
+
+.. note::
+   The product types listed above are part of the standard *Inventory* app. For access to the fields
+   below, install the corresponding apps **in addition** to *Inventory*.
+
+   To install additional applications, navigate to the :menuselection:`Apps` application, and select
+   the :guilabel:`Activate` button to install the desired apps or modules.
```
Suggestion to replace install instructions with a simple reference :)
```suggestion
   below, :ref:`install <general/install>` the corresponding apps **in addition** to *Inventory*.
```

samueljlieber commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,236 @@
+   the :guilabel:`Activate` button to install the desired apps or modules.
+
+- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Appointment* app.
+  Requires the installation of the *Calendar* app and :guilabel:`Pay to Book` module
```
The module name can follow the same format (italics) as an app name :) also suggesting to add the technical name of the module for specificity  
```suggestion
  Requires the installation of the *Calendar* app and *Pay to Book* (`appointment_account_payment`)
  module
```

Felicious replied:
Great point!

samueljlieber commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,236 @@
+Packages
+--------
+
+Both storable and consumable products can be placed in packages.
+
+.. seealso::
+   :doc:`package`
```
Suggestion to use a reference instead of a seealso here only because I feel the seealso breaks the flow of this section, while a reference preserves it. Up to you!
```suggestion
Both storable and consumable products can be placed in :doc:`packages <package>`.
```

samueljlieber reviewed (approved):
Hi @Felicious, awesome job on this PR! I only have a few suggestions for your consideration and two quick corrections. Can you please update the commit message to have a space between the commit tag and app name?
```
[ADD] inventory: product type
```
I am approving now, since all of my comments (besides the module italics formatting) are optional. Feel free to pass this PR to ZST once the commit message and rst is updated! Thank you :)

Felicious commented:
@StraubCreative this is ready for you to merge!

StraubCreative commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,231 @@
+    :align: center
+    :alt: Set a product type on the product form.
+
+.. note::
+   The product types listed above are part of the standard *Inventory* app. For access to the fields
+   below, :ref:`install <general/install>` the corresponding apps **in addition** to *Inventory*.
```
The ref is not formatting in the HTML here. Did you mean to link to [this doc](https://www.odoo.com/documentation/17.0/applications/general/apps_modules.html)?

Felicious replied:
I'm not sure what you mean by "not formatting in the HTML".

This is the rendered result I see:
![image](https://github.com/odoo/documentation/assets/23611711/2700078f-c5ae-46aa-ae3f-70bac27ec634)

And when I click on the link, I go to:
![image](https://github.com/odoo/documentation/assets/23611711/739ff09f-479a-405a-8290-9162c5e12b49)

I do intend to link to the doc you listed, but specifically, the "Install apps and modules" section

![image](https://github.com/odoo/documentation/assets/23611711/8676e0ea-08eb-43b4-9868-68ad7ba2ee06)

StraubCreative commented on content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst:
```diff
@@ -0,0 +1,231 @@
+   The product types listed above are part of the standard *Inventory* app. For access to the fields
+   below, :ref:`install <general/install>` the corresponding apps **in addition** to *Inventory*.
+
+- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Appointment* app.
```
missing an `s` here 😉 
```suggestion
- :guilabel:`Booking Fees`: charge a fee for booking appointments through the *Appointments* app.
```

StraubCreative reviewed (approved):
Hi @Felicious 
This is a great doc that super clearly explains product types. I'm sure customers (and support) will be very happy to have it!

I have just a few items for you below to take a look at, after which, we can merge.

There are a few necessary changes, however, for the optional ones— please address at your discretion. Can merge after, when ready, thanks!
.....
@robodoo delegate=Felicious

Felicious commented:
@robodoo r+
